### PR TITLE
Update citation table with new types

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/greenelab/meta-review/issues/111

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -121,12 +121,21 @@ Manual CSL JSON also supports references without standard identifiers, such as p
 | Digital Object Identifier (DOI) | DOI [Content Negotiation](https://citation.crosscite.org/docs.html) | `doi:10.1098/rsif.2017.0387` | [@doi:10.1098/rsif.2017.0387] |
 | PubMed Identifier (PMID) | NCBI [E-utilities](https://www.ncbi.nlm.nih.gov/books/NBK25501/) | `pmid:25851694` | [@pmid:25851694] |
 | PubMed Central Identifier (PMCID) | NCBI [Literature Citation Exporter](https://api.ncbi.nlm.nih.gov/lit/ctxp/) | `pmcid:PMC4719068` | [@pmcid:PMC4719068] |
-| arXiv identifier | [arXiv API](https://arxiv.org/help/api/index) | `arxiv:1502.04015v1` | [@arxiv:1502.04015v1] |
-| URL | [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1] | `url:https://lgatto.github.io/open-and-open/` | [@url:https://lgatto.github.io/open-and-open/] |
+| arXiv ID | [arXiv API](https://arxiv.org/help/api/index) | `arxiv:1502.04015v1` | [@arxiv:1502.04015v1] |
+| ISBN | [Zotero translation-server](https://github.com/zotero/translation-server) | `isbn:9780262517638` | [@isbn:9780262517638] |
```
Comment (agitter) [inline content/02.main-text.md]: For the other identifiers we spell out the acronyms.  Should we use `International Standard Book Number (ISBN)`?

The example `@isbn:9780262517638` is okay, but if you want to reuse an existing book citation, we could use [Opening Science](https://link.springer.com/book/10.1007%2F978-3-319-00026-8#about) as the example here.  We currently cite it as `doi:10.1007/978-3-319-00026-8`.

File: content/02.main-text.md [hunk 2]
```diff
@@ -121,12 +121,21 @@ Manual CSL JSON also supports references without standard identifiers, such as p
...
+| URL | Zotero translation-server | `url:https://lgatto.github.io/open-and-open/` | [@url:https://lgatto.github.io/open-and-open/] |
+| Wikidata ID | Zotero translation-server | `wikidata:Q56458321` | [@wikidata:Q56458321] |
```
Comment (agitter) [inline content/02.main-text.md:127]: Nice example

File: content/02.main-text.md [hunk 3]
```diff
@@ -121,12 +121,21 @@ Manual CSL JSON also supports references without standard identifiers, such as p
...
+| Raw | Provided by user | `raw:dongbo-conversation` | [@raw:dongbo-conversation] |
```
Comment (agitter) [inline content/02.main-text.md:128]: We should probably add Dongbo to the acknowledgements as well.

File: content/02.main-text.md [hunk 4]
```diff
@@ -121,12 +121,21 @@ Manual CSL JSON also supports references without standard identifiers, such as p
...
 | Tag | Source for tagged identifier | `tag:avasthi-preprints` | [@tag:avasthi-preprints] |
 
-Table: Citation types supported by Manubot.
-Authors may optionally map a named tag to one of the other supported identifier types.
+Table: **Citation types supported by Manubot.**
+Manubot allows users to cite different types of persistent identifiers, as shown in this table.
+Metadata source indicates the primary resource used to retrieve bibliographic metadata.
+For certain identifier types, additional metadata sources are queried should the primary fail.
+For example, When translation-server ISBN lookup fails, Manubot tries Wikipedia's [Citoid](https://www.mediawiki.org/wiki/Citoid) service followed by the [isbnlib](https://github.com/xlcnd/isbnlib) Python package.
```
Comment (agitter) [inline content/02.main-text.md]: `when`

File: content/02.main-text.md [hunk 5]
```diff
@@ -121,12 +121,21 @@ Manual CSL JSON also supports references without standard identifiers, such as p
...
+When translation-server URL lookup fails, Manubot then tries [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1].
+Raw citations enable citing works when no supported persistent identifiers exist, but require that user specifies the metadata.
```
Comment (agitter) [inline content/02.main-text.md]: `that the user`

**Review by agitter — CHANGES_REQUESTED** [review]
Great updates.  The expanded table caption is especially useful.

File: content/99.back-matter.md [hunk 6]
```diff
@@ -4,6 +4,7 @@ We would like to thank the authors of the Deep Review who helped us test collabo
 The authors who responded favorably to being acknowledged are Paul-Michael Agapow, Amr M. Alexandari, Brett K. Beaulieu-Jones, Anne E. Carpenter, Travers Ching, Evan M. Cofer, Dave DeCaprio, Brian T. Do, Enrico Ferrero, David J. Harris, Michael M. Hoffman, Alexandr A. Kalinin, Anshul Kundaje, Jack Lanchantin, Christopher A. Lavender, Benjamin J. Lengerich, Zhiyong Lu, Yifan Peng, Yanjun Qi, Gail L. Rosen, Avanti Shrikumar, Srinivas C. Turaga, Gregory P. Way, Laura K. Wiley, Stephen Woloszynek, Wei Xie, Jinbo Xu, and Michael Zietz.
 In addition, we thank Ogun Adebali, Evan M. Cofer, and Robert Gieseke for contributing to the Manubot template manuscript.
 We are grateful for additional Manubot discussion and testing by Alexander Dunkel, Ansel Halliburton, Achintya Rao, and other GitHub users.
+Setup and maintainance of the Zotero translation-server for Manubot usage was performed by Dongbo Hu.
```
Comment (dhimmel) [inline content/99.back-matter.md:7]: @dongbohu are you OKAY with this acknowledgement?

File: content/02.main-text.md (hunk 1 above)
Comment (dhimmel) [reply content/02.main-text.md]: In https://github.com/greenelab/meta-review/pull/113/commits/8b9d518e2c12002a2153221d62a73f7fd4125eee, I cited the "Open Access Book" for the following sentence:

> For example, open access publishing provides an economic model that encourages maximal dissemination and reuse of scholarly articles

Therefore, it's no longer just used by this table.

**Comment by dongbohu** [comment]
Looks good. Thanks.

**Review by agitter — APPROVED** [review]

**Comment by agitter** [comment]
There are extra commas in the author name for the new ISBN citation
image

This may not be severe enough to require a manual override though.

**Pull request merged**